lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/classify/DiscreteClassifier.java

```diff
@@ -0,0 +1,115 @@
+package edu.stanford.futuredata.macrobase.analysis.classify;
+
+import edu.stanford.futuredata.macrobase.analysis.classify.stats.WeightedPercentile;
+import edu.stanford.futuredata.macrobase.datamodel.DataFrame;
+
+/**
+ * Classify rows by high / low values based on the group mean.
+ * Returns a new dataframe with a column representation of the estimated number of outliers
+ * for each group, which will either be zero or all elements in the group.
+ */
+public class DiscreteClassifier extends CubeClassifier implements ThresholdClassifier {
```

Comment:
No test and it isn't used anywhere, might be worth deleting this or adding test and actually making it possible to use from CubeClassifier